packages/components/src/Button/Button.types.ts

```diff
@@ -45,24 +45,6 @@ interface ButtonLabelProps extends ButtonNonComposableProps {
   readonly label: string;
 }
 
-type BaseButtonProps = XOR<
```

Comment:
Thanks for cleaning up this. For reference the types were accidentally duplicated and this just removes the extras that aren't needed